membership-attribute-service/app/services/AttributesFromZuora.scala

```diff
@@ -26,6 +26,9 @@ import services.AttributesMaker.getSubsWhichIncludeDigitalPack
 import utils.FutureRetry._
 
 class AttributesFromZuora(implicit val executionContext: ExecutionContext, system: ActorSystem) extends LoggingWithLogstashFields {
```

Comment:
there is only one AttributesFromZuora created, here https://github.com/guardian/members-data-api/blob/44e10cc002f3a4b8211200c89b7268088d80c7f6/membership-attribute-service/app/wiring/AppLoader.scala#L57